src/main/java/org/mastodon/mamut/launcher/RecentProjectsPanel.java

```diff
@@ -140,4 +154,44 @@ private void remakeGUI( final Consumer< String > projectOpener )
 		revalidate();
 		repaint();
 	}
+
+
+	public static final class MouseDblClickOpenPath implements MouseListener {
+		final String Uri;
+		public MouseDblClickOpenPath(final String pathOnDblClick) {
+			Uri = Paths.get(pathOnDblClick).getParent().toUri().toString();
```

Comment:
action no. 2: here is the dedicated mouse listener created, it memorizes a path for its later acting

this path is a **parent** path to the given one... so it opens

- a folder which holds the `.mastodon` file
- a folder which holds the project folder.. (so not the project's content but a folder with the project's folder)